.zenodo.json

```
@@ -6,18 +6,23 @@
       "orcid":"0000-0003-3396-6154"
     },
     {
-      "affiliation":"HZDR",
-      "name":"Bastrakova, Kseniia"
-    },
-    {
-      "name":"Marre, Brian Edward",
-      "affiliation":"HZDR, TU Dresden"
+      "affiliation":"HZDR, TU-Dresden",
+      "name":"Carstens, Finn-Ole"
     },
     {
       "affiliation":"HZDR",
       "name":"Debus, Alexander",
       "orcid":"0000-0002-3844-3697"
     },
+    {
+      "affiliation":"HZDR",
```

```suggestion
      "affiliation":"HZDR, TZ-Dresden",
```

Sorry, I did not see that before